packages/databyss-services/interfaces/Block.ts

```diff
@@ -2,6 +2,13 @@ import { BlockRelation } from '@databyss-org/editor/interfaces/index'
 import { SelectOption } from './UI'
 import { Text } from './Text'
 
+export enum DocumentType {
```

Comment:
See note above about `DocumentType` scope. This should be in `@databyss-org/data`